### ISSUE TITLE
[Bug]: Iced Bar turns black in full screen mode

### ISSUE THREAD

**Issue by ReagentX** [jordanbaird/Ice#254]
### Check Existing Issues

- [X] I have searched existing issues, and this issue is not a duplicate

### Description

Image: 

<img width="436" alt="image" src="https://github.com/user-attachments/assets/0b338002-0604-4713-9709-a6c7583ea0b5">

Video: 

https://github.com/user-attachments/assets/3d89bb29-4da7-4354-9657-d3eff7f1c9ae

### Steps to Reproduce

1. Open an app in full screen mode 
2. Activate the Ice Bar

### Ice Version

0.10.2

### macOS Version

14.5 (23F79)

**Comment by jordanbaird** [comment]
I can't reproduce this exact case, but images disappearing from the Ice Bar is definitely a known issue. For some reason, the screen capture API we're using fails under certain circumstances. If you keep your mouse over the menu bar so that it doesn't disappear, do the images stay there?

**Comment by ReagentX** [comment]
Yeah, they only seem to disappear when I mouse away and the menubar hides itself. This is on a laptop with a notch.

**Comment by mmuejde** [comment]
Same problem here

![CleanShot 2024-08-16 at 12 20 17](https://github.com/user-attachments/assets/77129519-0b3a-4e42-ac4d-1da7fbdedc73)


MacOS: 14.5 (23F79)
